ynr/apps/elections/uk/templates/candidates/person-view.html

```diff
@@ -62,21 +48,12 @@
 
 <div class="person__hero">
   <div class="person__photo">
-    {% if person.primary_image %}
-      {% thumbnail person.primary_image "x80" as im %}
-        <img class="person-avatar" src="{{ im.url }}"/>
-      {% endthumbnail %}
-    {% else %}
-      {% if person.gender|lower == 'female' %}
-        <img class="person-avatar" src="{% static 'candidates/img/blank-woman.png' %}"/>
-      {% else %}
-        <img class="person-avatar" src="{% static 'candidates/img/blank-man.png' %}"/>
-      {% endif %}
-      {% if user.is_authenticated %}
-        <a class="upload-photo" href="{% url 'photo-upload' person.id %}">
-          {% trans 'Upload photo' %}
-        </a>
-      {% endif %}
+
+    <img class="person-avatar" src="{{ person.get_display_image_url }}"/>
+    {% if not person.primary_image and user.is_authenticated %}
+      <a class="upload-photo" href="{% url 'photo-upload' person.id %}">
+      w  {% trans 'Upload photo' %}
```

Comment:
What is the significance of the 'w' character here?

Reply:
Whoops!